src/containers/CheckoutPage/CheckoutPageWithPayment.js

```diff
@@ -506,6 +511,12 @@ export const CheckoutPageWithPayment = props => {
     orderData?.deliveryMethod === 'shipping' &&
     !hasTransactionPassedPendingPayment(existingTransaction, process);
 
+  const listingLocation = listing?.attributes?.publicData?.location;
+  const isBooking = processName === BOOKING_PROCESS_NAME;
+  const isPurchase = processName === PURCHASE_PROCESS_NAME;
+  const showPickUpLocation = isPurchase && orderData?.deliveryMethod === 'pickup';
+  const showLocation = isBooking && listingLocation?.address;
```

Comment:
For symmetry, would it make sense to name this showBookingLocation, since it's specifically on bookings? Or is this something that's going to eventually be applied to negotiation location as well?